cmd/kubelet/app/options/options.go

```diff
@@ -98,6 +98,7 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.DurationVar(&s.HTTPCheckFrequency.Duration, "http-check-frequency", s.HTTPCheckFrequency.Duration, "Duration between checking http for new data")
 	fs.StringVar(&s.ManifestURL, "manifest-url", s.ManifestURL, "URL for accessing the container manifest")
 	fs.StringVar(&s.ManifestURLHeader, "manifest-url-header", s.ManifestURLHeader, "HTTP header to use when accessing the manifest URL, with the key separated from the value with a ':', as in 'key:value'")
+	fs.StringSliceVar(&s.ManifestURLHeaders, "manifest-url-headers", s.ManifestURLHeaders, "Comma-separated list of HTTP headers to use when accessing the manifest URL, with the key separated from the value with a ':', as in 'key:value', will be combined with ManifestURLHeader")
```

Comment:
I think we should deprecate `ManifestURLHeader` in favor of `ManifestURLHeaders` and mark the `manifest-url-header` flag as deprecated with `fs.MarkDeprecated(...)`. It seems kind of redundant to keep both of these around for an extended period of time, since the new flag can accomplish the job of the old flag (ofc the old flag still needs to work for a few releases for backwards compatibility reasons).

@mikedanese what do you think?